### DISCUSSION TITLE
How to Install MediaCMS using CPanel

### DISCUSSION THREAD

**Discussion by beade** [mediacms-io/mediacms#468] [Q&A]
I'm just trying to install MediaCMS on my hosting using CPanel.
Can you help me with that? Do you have any video tutorials or instructions manuals (For hosting CPanel)? I really need to install and configure MediaCMS. I've already downloaded the folder of your GitHub but I'm confused about what to do next.
Thanks in advance for your help.

**Comment by netzih** [comment]
Yes,
Or using plesk (I would assume the installation instructions would be the same).
I am sure it would be using docker but I tried setting it up but I am getting errors when trying to configure it.
